cpp/serve/config.h

```diff
@@ -230,6 +230,10 @@ class EngineConfigNode : public Object {
   /*! \brief The number of tokens to generate in speculative proposal (draft). */
   int spec_draft_length = 4;
 
+  /*************** Jump Forward ***************/
+  /*! \brief Whether to disable the jump-forward decoding. Mainly for test purposes. */
+  bool debug_disable_jump_forward;
```

Comment:
Should this be a per-request field in DebugConfig? https://github.com/mlc-ai/mlc-llm/blob/203cda6b5227407b614c24cc4965527de5b6ccfe/cpp/serve/config.h#L50-L54

Reply:
That's reasonable